proposals/2020/Reporting Weather Data.md

```diff
@@ -0,0 +1,104 @@
+# Reporting Weather Data
+
+## Overview
+
+This proposal is to improve the reporting of weather data by making a weather station an "entity" which is referencable from a time series element.
+
+## Justification
+
+BuildingSync is used with tools which require or report weather data. BuildingSync needs a way to report time series data for temperature (and other weather qualities) and link that to a weather station. Currently you can report time series weather data using `TimeSeriesReadingQuantity`, but you can't link it to weather station. It would be valuable (and consistent) to be able to link a `auc:TimeSeries` to a weather station.
+
+## Implementation
+
+The set of elements
+
+- auc:WeatherDataStationID
+- auc:WeatherStationName
+- auc:WeatherStationCategory
+
+are currently available under Site and Building, so that's where we will put the new weather station element. Note that we will **not** remove the existing set of elements, but they will become _mutually exclusive_ with the new `auc:WeatherStations` element until they are deprecated in the next major version (3.0).
+
+Schema:
+```xml
+<!-- Weather station schema -->
+<xs:element name="WeatherStations" minOccurs="0">
+    <xs:complexType>
+        <xs:sequence>
+            <xs:element name="WeatherStation" maxOccurs="unbounded">
+                <xs:complexType>
+                    <xs:sequence>
+                        <xs:element ref="auc:WeatherDataStationID" minOccurs="0"/>
+                        <xs:element ref="auc:WeatherStationName" minOccurs="0"/>
+                        <xs:element ref="auc:WeatherStationCategory" minOccurs="0"/>
+                    </xs:sequence>
+                    <xs:attribute name="ID" type="xs:ID" use="required"/>
+                </xs:complexType>
+            </xs:element>
+        </xs:sequence>
+    </xs:complexType>
+</xs:element>
+
+<!-- TimeSeries schema -->
+<xs:complexType name="TimeSeriesType">
+    <xs:sequence>
+        <!-- ... other sub elements ... -->
+        <xs:element name="WeatherStationID" minOccurs="0">
+            <xs:annotation>
+                <xs:documentation>ID number of weather station this time series contributes to.</xs:documentation>
```

Comment:
Would like some feedback on "WeatherStationID" as reference. This _might_ be confusing to users, e.g. the might think of the NOAA station ID here. Though this naming is consistent with our other ID reference elements. Is there something we should add to the documentation to make this more clear?

Reply:
Without this proposal, is it possible to link a time series to a non-`auc:WeatherStations` weather station of a specific site or building (i.e., specified by `auc:WeatherDataStationID`, etc.)?

With this proposal, is it possible to link a time series to a non-`auc:WeatherStations` weather station of a specific site or building?

Reply:
(1) -- No, it's not possible currently. You can provide weather station information _inside_ of the scenario (see auc:WeatherType, but there's no explicit link)

(2) -- No, you are not able to link to non-`auc:WeatherStation`s in this proposal

So it's not _really_ a concern about creating a valid doc, its just a concern about how it might be interpreted by a user